core/tests/org.fusesource.ide.camel.validation.tests.integration/src/org/fusesource/ide/camel/validation/xml/AbstractXMLCamelRouteValidorTestHelper.java

```diff
@@ -28,12 +28,13 @@
 import org.fusesource.ide.camel.model.service.core.io.CamelIOHandler;
 import org.fusesource.ide.camel.model.service.core.model.CamelFile;
 import org.fusesource.ide.camel.model.service.core.tests.integration.core.io.FuseProject;
+import org.fusesource.ide.camel.model.service.core.util.CamelCatalogUtils;
 import org.junit.Rule;
 
 public class AbstractXMLCamelRouteValidorTestHelper {
 
 	@Rule
-	public FuseProject fuseProject = new FuseProject(getClass().getSimpleName());
+	public FuseProject fuseProject = new FuseProject(getClass().getSimpleName(), CamelCatalogUtils.CAMEL_VERSION_LATEST_COMMUNITY);
```

Comment:
Kind of surprised we weren't already going against the latest community version here, but good catch

Reply:
when not precising we were going against latest productized version which is too old until Fuse 7 will be released.